src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/tests/test_resource.py

```diff
@@ -432,50 +432,25 @@ def test_group_deployment_thru_uri(self, resource_group):
         self.cmd('group deployment list -g {}'.format(self.resource_group), checks=NoneCheck())
 
 
-class ResourceMoveScenarioTest(VCRTestBase):
-    def __init__(self, test_method):
-        super(ResourceMoveScenarioTest, self).__init__(__file__, test_method)
-        self.source_group = 'res_move_src_group'
-        self.destination_group = 'res_move_dest_group'
+class ResourceMoveScenarioTest(ScenarioTest):
+    @ResourceGroupPreparer(name_prefix='cli_test_resource_move_dest', parameter_name='resource_group_dest')
+    @ResourceGroupPreparer(name_prefix='cli_test_resource_move_source')
+    def test_resource_move(self, resource_group, resource_group_dest):
+        nsg1_name = self.create_random_name('nsg-move', 20)
+        nsg2_name = self.create_random_name('nsg-move', 20)
 
-    def test_resource_move(self):
-        self.execute()
+        nsg1 = self.cmd('network nsg create -n {} -g {}'.format(nsg1_name, resource_group))
```

Comment:
You can get the JSON here immediately since you're not using the `ExecutionResult`'s other properties.